Library/Invoice.cs

```diff
@@ -104,6 +105,16 @@ public byte[] GetPdf(string acceptLanguage = "en-US")
         }
 
         /// <summary>
+        /// Returns a PDF representation of an invoice
+        /// </summary>
+        /// <param name="acceptLanguage">Language for invoice, defaults to en-US.</param>
+        /// <returns></returns>
+        public async Task<byte[]> GetPdfAsync(string acceptLanguage = "en-US")
```

Comment:
It appears that the commit I was expecting to release as a pull request after the one for updating the tests snuck into this PR somehow. The sneaky commit is in a finished state and only has 5 files changed which are adding additional endpoints. 

Would you prefer that I commit a revert and submit a new PR? I have no problem either way.

Thanks.